client/public/logout.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>ANET Logout</title>
+    <style>
+        * {
+            margin: 0;
+            padding: 0;
+            box-sizing: border-box;
+        }
+        body {
+            height: 98vh;
+            display: grid;
+            place-content: center;
+            background-color: #f2f2f2;
+            font-family: -apple-system, BlinkMacSystemFont, “Segoe UI”, Roboto,
+                "Helvetica Neue", Helvetica, Arial, sans-serif;
+        }
+        .logout-container {
+            height: 100%;
+            width: 100%;
+            border-radius: 6px;
+            background-color: #ffffff;
+        }
+        .content-container {
+            width: 550px;
+            height: 250px;
+            padding: 20px;
+            display: flex;
+            align-items: center;
+            flex-direction: column;
+            justify-content: space-between;
+        }
+        img { width: 100% }
+        button {
+            color: #fff;
+            border: none;
+            font-size: 20px;
+            border-radius: 6px;
+            padding: 10px 20px;
+            background-color: #0071bc
```

Comment:
Missing trailing semicolon. Probably add `cursor: pointer;` here (so user sees that the button can be clicked).